# Author: Ehsan Moravveji (VSC, KU Leuven)
# Purpose: to prevent the following error message:
#          "pdm.backend.exceptions.ConfigError: 
#           Cannot find the version from SCM or SCM isn't detected."

diff -ruN nipype-1.9.2-orig/pyproject.toml nipype-1.9.2/pyproject.toml
--- nipype-1.9.2-orig/pyproject.toml	2025-03-07 13:15:12.132810000 +0100
+++ nipype-1.9.2/pyproject.toml	2025-03-07 13:17:04.736965000 +0100
@@ -25,3 +25,6 @@
 env = "PYTHONHASHSEED=0"
 filterwarnings = ["ignore::DeprecationWarning"]
 junit_family = "xunit2"
+
+[tool.setuptools_scm]
+write_to = 'nipype/_version.py'
